### PR TITLE
User/krschau/fix ssh widget crash

### PR DESCRIPTION
**Pull request by krschau**
## PR checklist
- [ ] Closes #xxx
- [ ] Tests added/passed
- [ ] Documentation updated

### CODE REVIEW

File: tools/Dashboard/DevHome.Dashboard/ViewModels/WidgetViewModel.cs [hunk 1]
```diff
@@ -312,42 +310,36 @@ private Grid GetErrorCard(string error, string subError = null)
         return grid;
     }
 
-    private JObject WrapJsonString(string jsonString)
+    private Newtonsoft.Json.Linq.JObject WrapJsonString(string jsonString)
```
Comment (dhoehna) [inline tools/Dashboard/DevHome.Dashboard/ViewModels/WidgetViewModel.cs:313]: Does `using Newtonsoft.Json.Linq` throw an error?

File: tools/Dashboard/DevHome.Dashboard/ViewModels/WidgetViewModel.cs [hunk 2]
```diff
@@ -312,42 +310,36 @@ private Grid GetErrorCard(string error, string subError = null)
...
-        try
+        if (inputsObject != null)
         {
-            objB = JObject.Parse(jsonStringB);
-        }
-        catch (JsonReaderException)
-        {
-            objB = WrapJsonString(jsonStringB);
+            var inputType = inputsObject.ValueType;
+            if (inputType != Windows.Data.Json.JsonValueType.Null)
```
Comment (dhoehna) [inline tools/Dashboard/DevHome.Dashboard/ViewModels/WidgetViewModel.cs:339]: Is line 339, 340, 341, and 342 tabbed with "\t"?

File: tools/Dashboard/DevHome.Dashboard/ViewModels/WidgetViewModel.cs [hunk 3]
```diff
@@ -312,42 +310,36 @@ private Grid GetErrorCard(string error, string subError = null)
...
+                objA = Newtonsoft.Json.Linq.JObject.Parse(actionValue.Stringify());
+            }
+            else if (dataType == Windows.Data.Json.JsonValueType.String)
+            {
+                objA = WrapJsonString(actionValue.Stringify());
+            }
         }
 
-        try
+        if (inputsObject != null)
```
Comment (dhoehna) [inline tools/Dashboard/DevHome.Dashboard/ViewModels/WidgetViewModel.cs:336]: NIT: inputsObject?.ValueType != Windows.Data.Json.JsonValueType.Null

File: tools/Dashboard/DevHome.Dashboard/ViewModels/WidgetViewModel.cs (hunk 1 above)
Comment (krschau) [reply tools/Dashboard/DevHome.Dashboard/ViewModels/WidgetViewModel.cs:313]: Nope, that's just the namespace JObject lives under.

File: tools/Dashboard/DevHome.Dashboard/ViewModels/WidgetViewModel.cs (hunk 2 above)
Comment (krschau) [reply tools/Dashboard/DevHome.Dashboard/ViewModels/WidgetViewModel.cs:339]: No, I think the color in GitHub comes from it being indented one more time.

**Pull request merged**